### DISCUSSION TITLE
Network stacked on the block 223584

### DISCUSSION THREAD

**Discussion by supervitek** [Chia-Network/chia-blockchain#7093] [Support]
Hi, all the nodes are stucked on that height. Any idea what to do next?

**Comment by loppefaaret** [comment]
you might have to switch to INFO level logging, to see what the full_node is doing, that is preventing it from getting any further ?
`chia configure --set-log-level INFO`
and restart chia
that will allow INFO level logging into the `debug.log` file, located under you users folder `~/.chia/mainnet/log/`

**Comment by BrandtH22** [comment]
Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
